docs/_docs/windows.md

```diff
@@ -20,6 +20,9 @@ For a more conventional way of installing Jekyll you can follow this [complete g
 
 [windows-installjekyll3]: https://labs.sverrirs.com/jekyll/
 
+\[Update 12/23/2016\]: Updates in the infrastructure of Ruby may cause SLL errors when attemptying to use gem install on a version of ruby older than 2.6. (The ruby package installed via the Chocolatey tool is version 2.3) If you have installed an older version, you can update ruby using the directions [here.][ssl-certificate-update]
```

Comment:
Suggesting a couple of edits : 
  - I don't think including a date is necessary. The information is not fresh-off-the-press and it is understandable by only mentioning the version-number.
  - its **SSL** instead of **SLL**; 
  - **gem install** should be within backticks.: `gem install`
  - its not "...***ruby** older than 2.6*" but "...***RubyGems Package** older than 2.6*." 
  - again, "**The RubyGems package** installed via the Chocolatey.."
  - ...update **RubyGems package** using the directions [here]..."